src/main/resources/templates/query.html

```diff
@@ -522,3 +1020,4 @@ <h2 id="medical_record">Medical Record</h2>
 
 
 </html>
+>>>>>>>>> Temporary merge branch 2
```

Comment:
??

Reply:
merging issue? I guess IntelliJ did that